src/limits/MemoryLimitListener.h

```diff
@@ -5,6 +5,13 @@
 #include "seccomp/policy/SyscallPolicy.h"
 #include "tracer/TraceEventListener.h"
 
+#include <sys/mman.h>
+
+#ifndef MREMAP_DONTUNMAP
+//Due to this flag being introduced in Linux 5.7 there are many system which do not define it
+#define MREMAP_DONTUNMAP 4
+#endif
+
```

Comment:
If this is only required inside the `SeccompRule` in MemoryLimitListener.cc, maybe instead of having this include here in  MemoryLimitListener.h, maybe we can have it near the top of MemoryLimitListener.cc, together with other `sys/*` includes?